src/controllers/rootController/rootController.ts

```diff
@@ -3,10 +3,27 @@
 // and another maybe for api/download which maybe is called DownloadController and allows the user to download the processed video
 
 import express from 'express';
```

Comment:
I know I did this, but could we use destructured imports like `import { Application, Request, Response } from 'express'` rather than `import express from 'express'`